modules/navigation/navigation_mesh_generator.cpp

```diff
@@ -194,10 +216,11 @@ void NavigationMeshGenerator::_parse_geometry(Transform p_accumulated_transform,
 
 					BoxShape *box = Object::cast_to<BoxShape>(*s);
 					if (box) {
-						Ref<CubeMesh> cube_mesh;
-						cube_mesh.instance();
-						cube_mesh->set_size(box->get_extents() * 2.0);
-						mesh = cube_mesh;
+						Array arr;
```

Comment:
Shouldn't this be the length of the vertices?

Reply:
we don't know the number of vertices at this point so we're resizing to `VS::ARRAY_MAX`